serverless.yml

```diff
@@ -88,22 +88,40 @@ functions:
 
 # =--= MISC. AWS RESOURCES =-------------------------------------------------------------=
 
-# resources:
-#   Resources:
-#     usersTable:
-#       Type: AWS::DynamoDB::Table
-#       Properties:
-#         TableName: 52calls-52weeks-users
-#         KeySchema:
-#           - AttributeName: id
-#             KeyType: HASH
-#         AttributeDefinitions:
-#           - AttributeName: id
-#             AttributeType: S
-#         ProvisionedThroughput:
-#           ReadCapacityUnits: 5
-#           WriteCapacityUnits: 5
-#
+resources:
+  Resources:
+    usersTable:
+      Type: AWS::DynamoDB::Table
+      Properties:
+        TableName: 52calls-52weeks-users
+        KeySchema:
+          - AttributeName: email
+            KeyType: HASH
+          - AttributeName: district
+            KeyType: RANGE
+        AttributeDefinitions:
+          - AttributeName: email
+            AttributeType: S
+          - AttributeName: district
+            AttributeType: S
+        ProvisionedThroughput:
+          ReadCapacityUnits: 5
+          WriteCapacityUnits: 5
+
+    repsTable:
+      Type: AWS::DynamoDB::Table
+      Properties:
+        TableName: 52calls-52weeks-representatives
+        KeySchema:
+          - AttributeName: district
+            KeyType: HASH
+        AttributeDefinitions:
+          - AttributeName: district
+            AttributeType: S
+        ProvisionedThroughput:
+          ReadCapacityUnits: 5
+          WriteCapacityUnits: 5
```

Comment:
The impression I got from chatting with Seth yesterday was that you can overprovision bandwidth without getting linear correlation in your costs as long as you're not saturating it. Apparently there's a cost estimator that can help with this, but if that's the case I'd lean toward going higher on this and cutting the time of our batch uploads and reads in half. Although, grand scale, as long as we have queueing, shouldn't really matter, and we can easily reconfigure later with a redeploy